gallery/src/components/NxSubmitMask/NxSubmitMaskCustomMessageExample.tsx

```diff
@@ -17,8 +17,10 @@ export default function NxSubmitMaskCustomMessageExample() {
   }
 
   return (
-    <div onKeyUp={handleKeyUp} className="gallery-submit-mask-area">
-      <NxButton onClick={setMaskVisible.bind(null, true)}>Click here to create a NxSubmitMask</NxButton>
+    <div className="gallery-submit-mask-area">
+      <NxButton onKeyUp={handleKeyUp} onClick={setMaskVisible.bind(null, true)}>
```

Comment:
This keyup handler is kind of hacky, it's just to give the user a way to get out of the example.  When it was on the div, the plugin wanted the div to have a role.  I couldn't really think of one and it works just as well on the button, so...